src/components/ebay-menu/index.js

```diff
@@ -213,6 +215,8 @@ function processAfterStateChange(itemIndexes) {
         });
     }
 
+    elementScroll.scroll(itemEl);
```

Comment:
I think we should probably wrap this (and the one below) in `if (this.state.isRadio || this.state.isCheckbox)` so we avoid querying the DOM unnecessarily.